ci/test/lint-fast.sh

```diff
@@ -30,10 +30,14 @@ try cargo --locked about generate ci/deploy/licenses.hbs > /dev/null
 
 
 if [[ "${BUILDKITE_PULL_REQUEST:-true}" != "false" ]]; then
+  # see ./ci/test/lint-buf/README.md
+
   fetch_pr_target_branch
 
+  ci_collapsed_heading "Generate protobuf config"
+  try bin/pyactivate ./ci/test/lint-buf/generate-buf-config.py
```

Comment:
Consider uploading the yaml file that was produced as a buildkite artifact so that it can be examined visually.

If you want to further proof this, consider adding yamllint to the ci-builder image and then use it to validate the yaml produced. My concern here is that , given how yaml as a format is flaky at best, a broken annotation is going to somehow cause no linting to take place.

Reply:
✅ file upload
✅ yamllint